recipe/meta.yaml

```diff
@@ -1,43 +1,52 @@
 {% set version = "1.2.0" %}
 
-
 package:
-    name: tuiview
-    version: {{ version }}
+  name: tuiview
+  version: {{ version }}
 
 source:
-    fn: TuiView-{{ version }}.tar.gz
-    url: https://bitbucket.org/chchrsc/tuiview/downloads/TuiView-{{ version }}.tar.gz
-    sha256: 1340136787f29b1f9146cf4819f7d44fffc8887707c42e78c79b7c7c9f6ba4e2
+  fn: TuiView-{{ version }}.tar.gz
+  url: https://bitbucket.org/chchrsc/tuiview/downloads/TuiView-{{ version }}.tar.gz
+  sha256: 1340136787f29b1f9146cf4819f7d44fffc8887707c42e78c79b7c7c9f6ba4e2
 
 build:
-    number: 0
-    entry_points:
-        - tuiview = tuiview.viewerapplication:run
-        - tuiviewwritetable = tuiview.writetableapplication:run
+  number: 0
+  entry_points:
+    - tuiview = tuiview.viewerapplication:run
+    - tuiviewwritetable = tuiview.writetableapplication:run
 
 requirements:
-    build:
-        - python
-        - numpy x.x
-        - gdal 2.2.*
-        - pyqt 5.6.*
-    run:
-        - python
-        - numpy x.x
-        - gdal 2.2.*
-        - pyqt 5.6.*
+  build:
+    - python
+    - numpy 1.7.*  # [py27]
+    - numpy 1.9.*  # [py35]
+    - numpy 1.11.*  # [py36]
```

Comment:
This uses the oldest forwards compatible `numpy`. It is better than the `numpy x.x` syntax b/c it saves us CI resources and we don't need to rebuild the package when a new `numpy` release is out (unless the new `numpy` breaks the ABI :smile:).